### PR TITLE
Game Loop

### PR DESCRIPTION
**Pull request by grantdavis303**

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -22,7 +22,8 @@ def print_visual
     #{visual["A"][3]} | #{visual["B"][3]} | #{visual["C"][3]} | #{visual["D"][3]} | #{visual["E"][3]} | #{visual["F"][3]} | #{visual["G"][3]}
     #{visual["A"][2]} | #{visual["B"][2]} | #{visual["C"][2]} | #{visual["D"][2]} | #{visual["E"][2]} | #{visual["F"][2]} | #{visual["G"][2]}
     #{visual["A"][1]} | #{visual["B"][1]} | #{visual["C"][1]} | #{visual["D"][1]} | #{visual["E"][1]} | #{visual["F"][1]} | #{visual["G"][1]}
-    #{visual["A"][0]} | #{visual["B"][0]} | #{visual["C"][0]} | #{visual["D"][0]} | #{visual["E"][0]} | #{visual["F"][0]} | #{visual["G"][0]}"
+    #{visual["A"][0]} | #{visual["B"][0]} | #{visual["C"][0]} | #{visual["D"][0]} | #{visual["E"][0]} | #{visual["F"][0]} | #{visual["G"][0]}
+    "
```
Comment (grantdavis303) [inline lib/board.rb:26]: I accidentally committed the wrong message for this one - that's my bad. This one is just adding a space after displaying the board!

File: lib/board.rb [hunk 2]
```diff
@@ -21,13 +22,15 @@ def print_visual
...
```
Comment (wallacebilly1) [inline lib/board.rb:26]: Smart to add another line to provide some breaks in the visuals in the terminal.

File: lib/game.rb [hunk 3]
```diff
@@ -5,35 +5,54 @@ def initialize
...
     @message_bot = Message.new
+    @turns_taken = 0
+  end
+
+  def start
+    puts @message_bot.welcome
+    play = true   
+    answer = gets.chomp.downcase
+        if answer == 'q'
+            play = false
```
Comment (JaredMHobson) [inline lib/game.rb:16]: Ahhh 4 spaces

File: lib/game.rb [hunk 4]
```diff
@@ -5,35 +5,54 @@ def initialize
...
-    @board.print_visual
-
-    # once input is placed, checks for win conditions
+  end
 
-    # changes current_player
-    change_current_player
-    # loop    
-    take_turn
+  def player_take_turn
```
Comment (wallacebilly1) [inline lib/game.rb:44]: Nice job breaking up the take_turn methods.

**Review by wallacebilly1 — APPROVED** [review]
Great work here team!

File: lib/game.rb [hunk 5]
```diff
@@ -5,35 +5,54 @@ def initialize
...
+        else
+          puts @message_bot.invalid_command
+          answer = gets.chomp.downcase
+        end    
+    while play
+        puts @board.print_visual
+        take_turn
+        change_current_player
+        # Move later bros
+        puts @message_bot.last_piece_played(@board.last_piece_played)
```
Comment (JaredMHobson) [inline lib/game.rb:28]: Definitely like moving this under the current player message like Billy suggested

**Pull request merged**